### ISSUE TITLE
Cannot find module xxx

### ISSUE THREAD

**Issue by jessegood** [frankwallis/plugin-typescript#41]
I have a very simple config.js and react component:
```
/// <reference path="./../typings/react/react.d.ts" />
import * as React from 'react';
class Hello extends React.Component<{}, {}> {
    render() {
        return <p>Hello</p>
    }
}
React.render(<Hello />, document.body);
```

However, I'm getting this error in the console:
```
TypeScript "http://127.0.0.1:8080/src/main.tsx (4, 23):" logger.js!transpiled:16:13
TypeScript Cannot find module 'react'. (TS2307) logger.js!transpiled:16:13
```

Everything works fine (i.e. I see "Hello" in the browser), but I'm not sure why I get this error.
Is this "Working As Intended"?

**Comment by frankwallis** [comment]
No, it indicates that the TypeScript compiler cannot find the declaration of module "react". Does react.d.ts contain the statement ```declare module "react" {...}```?

The path to the react.d.ts file (./../typings) also looks a bit odd, I wonder if that could be causing the issue? Also - have you tried running this file through the command line compiler tsc?

**Comment by jessegood** [comment]
Thanks for the help. I found the problem...
I actually didn't show my entire `main.tsx`, this is the actually file:

```
// jspm-server
export let __hotReload = true

/// <reference path="./../typings/react/react.d.ts" />

import * as React from 'react';

class Hello extends React.Component<{}, {}> {
    render() {
        return <p>Hello</p>
    }
}
React.render(<Hello />, document.body);
```

I did not know this, but it appears `/// <reference path="./../typings/react/react.d.ts" />` must be the first line in the file. Moving it to the top solved the problem.

I figured this out looking at the transpiled js. If you do not put the reference path at the top, that line gets treated as a comment and removed...
